integration/command.go

```diff
@@ -285,6 +269,14 @@ func getInspektorGadgetLogs() string {
 	return sb.String()
 }
 
+// detectKubernetes checks if we are running in a Kubernetes environment
+func detectKubernetes() bool {
+	if _, err := os.Stat(filepath.Join(homedir.HomeDir(), "kube", "config")); err != nil {
+		return false
+	}
+	return true
+}
```

Comment:
Mmmm cc @alban?

Reply:
But if I am running local-gadget to debug a Kubernetes cluster, the test will try to print the IG logs, isn't it? If so, I think you can think about another solution for this in another PR to avoid blocking this one.

Reply:
I didn't test it, but do you think something like https://github.com/inspektor-gadget/inspektor-gadget/commit/52e7e35ee696289827637ef05076711b40b37d7e may work?

Reply:
This is looking like something we can work with. I will handle it like that!